### ISSUE TITLE
Inquiry About Training Loss for scSpace Intestinal Data Analysis

### ISSUE THREAD

**Issue by lxhbio** [ZJUFanLab/scSpace#4]
Dear ZJUFanLab Team,
I hope this message finds you well. I am a user of the scSpace tool. While following the example code for intestinal data in your repository (link: https://github.com/ZJUFanLab/scSpace/blob/master/vignettes/intestines.ipynb), I noticed that after training the model for 1000 epochs, the final Batch loss reached 233.05227661132812.

<img width="1853" height="1682" alt="Image" src="https://github.com/user-attachments/assets/7808155d-541d-44e1-b37a-b096a568b794" />

I have questions regarding the criteria for terminating training during my practical usage. Could you please advise if there is a clear standard loss threshold for the tool when processing similar intestinal data scenarios? Specifically, to what value should the loss decrease to indicate that the model training is sufficient and suitable for termination?
If you have any related parameter tuning suggestions or criteria for judging training convergence, I would greatly appreciate your guidance. Thank you for developing such a practical tool, and I look forward to your reply!
Sincerely,
Phd. xuhua
